ReactiveObjC/RACSequence.h

```diff
@@ -259,7 +259,7 @@ typedef RACSequence * _Nullable (^RACSequenceBindBlock)(ValueType _Nullable valu
 ///               return value must be an object. This argument cannot be nil.
 ///
 /// Returns a new sequence of reduced tuple values.
-- (RACSequence *)reduceEach:(id _Nullable (^)())reduceBlock;
+- (RACSequence *)reduceEach:(id _Nullable (^)(void))reduceBlock;
```

Comment:
This is an incorrect fix: this block does _not_ have `void` arguments. This is meant to be variadic block, so it's correct as-is.

The correct fix here is to disable the warning because the code is correct as written.